CharlieBackend.Business/Services/ScheduleServiceFolder/Helpers/SchedulesEventsDbEntityVerifier.cs

```diff
@@ -23,22 +25,34 @@ public async Task<string> ValidateCreateScheduleRequestAsync(CreateScheduleDto r
 
             if (!await _unitOfWork.StudentGroupRepository.IsEntityExistAsync(request.Context.GroupID))
             {
-                error.Append(" Group does not exist");
+                error.Append(ResponseMessages.NotExist("Group"));
             }
 
-            if (request.Context.MentorID.HasValue && !await _unitOfWork.MentorRepository.IsEntityExistAsync(request.Context.MentorID.Value))
+            if (request.Context.MentorID.HasValue && await _unitOfWork.MentorRepository.IsEntityExistAsync(request.Context.MentorID.Value))
             {
-                error.Append(" Mentor does not exist");
+                if ((await _unitOfWork.AccountRepository.GetAccountCredentialsById(request.Context.MentorID.Value)).IsActive == false)
+                {
+                    error.Append(" Mentor is not active");
```

Comment:
U expand class ResponseMessages, but used there usually string, why don't create such message in class?

Reply:
The corresponding message is added to the ResponseMessages class.